src/settings/appsettings.cpp

```diff
@@ -71,9 +71,14 @@ void AppSettings::setLocale(const QLocale &locale)
 
 void AppSettings::applyLocale(const QLocale &locale)
 {
+#ifdef Q_OS_DARWIN
+    const QString i18n_dir = QStringLiteral(".");
```

Comment:
Same about the case:
```suggestion
    const QString i18nDir = QStringLiteral(".");
```